src/main/java/frc/robot/commands/ColorWheelRotation.java

```diff
@@ -31,14 +34,15 @@ public void execute(){
                 previousColor = elevator.toWheelColor(colorSensor.getColor());
                 previousColor = currentColor;
             }
-            if (revCount >= Const.Elevator.NUMBER_OF_COLOR_CHANGES) {
+            /* Ask Bancino if this is correct inplace of "Const.Elevator.NUMBER_OF_COLOR_CHANGES" */
+            if (revCount >= Const.Elevator.elevatorColorChanges) {
```

Comment:
Remember that `elevatorColorChanges` is the variable you declared at the top of the file. It isn't in `Const`. This should be just `elevatorColorChanges`, *not* `Const.Elevator.elevatorColorChanges`.